osf/migrations/0177_update_schema_data.py

```diff
@@ -0,0 +1,285 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.13 on 2018-09-26 20:08
+from __future__ import unicode_literals
+
+import re
+import logging
+from django.db import migrations
+from osf.models.base import generate_object_id
+
+logger = logging.getLogger(__file__)
+
+# Dict to map original schema formats to new blocks types
+FORMAT_TYPE_TO_TYPE_MAP = {
+    ('multiselect', 'choose'): 'multi-select-input',
+    (None, 'multiselect'): 'multi-select-input',
+    (None, 'choose'): 'single-select-input',
+    ('osf-upload-open', 'osf-upload'): 'file-input',
+    ('osf-upload-toggle', 'osf-upload'): 'file-input',
+    ('singleselect', 'choose'): 'single-select-input',
+    ('text', 'string'): 'short-text-input',
+    ('textarea', 'osf-author-import'): 'contributors-input',
+    ('textarea', 'string'): 'long-text-input',
+    ('textarea-lg', None): 'long-text-input',
+    ('textarea-lg', 'string'): 'long-text-input',
+    ('textarea-xl', 'string'): 'long-text-input',
+}
+
+def remove_version_1_schemas(state, schema):
+    RegistrationSchema = state.get_model('osf', 'registrationschema')
+    assert RegistrationSchema.objects.filter(schema_version=1, abstractnode__isnull=False).count() == 0
+    assert RegistrationSchema.objects.filter(schema_version=1, draftregistration__isnull=False).count() == 0
+    RegistrationSchema.objects.filter(schema_version=1).delete()
+
+def update_schemaless_registrations(state, schema):
+    RegistrationSchema = state.get_model('osf', 'registrationschema')
+    AbstractNode = state.get_model('osf', 'abstractnode')
+
+    open_ended_schema = RegistrationSchema.objects.get(name='Open-Ended Registration')
+    open_ended_meta = {
+        '{}'.format(open_ended_schema._id): {
+            'summary': {
+                'comments': [],
+                'extra': [],
+                'value': ''
+            }
+        }
+    }
+
+    schemaless_regs_with_meta = AbstractNode.objects.filter(type='osf.registration', registered_schema__isnull=True).exclude(registered_meta={})
+    schemaless_regs_without_meta = AbstractNode.objects.filter(type='osf.registration', registered_schema__isnull=True, registered_meta={})
+
+    for reg in schemaless_regs_without_meta.all():
+        reg.registered_schema.add(open_ended_schema)
+        reg.registered_meta = open_ended_meta
+        reg.save()
+
+    for reg in schemaless_regs_with_meta.all():
+        reg.registered_schema.add(RegistrationSchema.objects.get(_id=reg.registered_meta.keys()[0]))
+
+def update_schema_configs(state, schema):
+    RegistrationSchema = state.get_model('osf', 'registrationschema')
+    for rs in RegistrationSchema.objects.all():
+        if rs.schema.get('description', False):
+            rs.description = rs.schema['description']
+            rs.save()
```

Comment:
update_schema_configs already existed as part of Matt's work, but now I'm mapping over the schema's description, instead of mapping this to config.tooltip.